functions/src/definitions/common/sendWhatsappMessage.js

```diff
@@ -77,7 +77,7 @@ async function sendWhatsappTemplateMessage(bot, to, templateName, languageCode =
     }),
   }
   const componentsArr = buttonComponentArr.concat(bodyComponentArr);
-  data = {
+  const data = {
```

Comment:
`data` is not defined before assignment, which is not allowed in strict mode, and causes code to crash

Reply:
You can try this by trying 
1: run `node` in CLI, and `test = 1`
2: run `node --use_strict` and `test = 1`

#2 will fail